content/docs/getting-started/servers.md

```diff
@@ -7,17 +7,18 @@ menu:
 weight: 110
 ---
 
-In this article, we will learn how to add "servers" to our AsyncAPI document. This is useful because it tells us where we should connect to send and receive messages.
-
 In the previous lesson, we learned how to create the definition of a simple [Hello World application](/docs/getting-started/hello-world), so let's take it from there.
 
+In this article, we will learn how to add "servers" to our AsyncAPI document. Adding and defining servers is useful because it specifies where and how to connect. The connection facilitates where to send and receive messages.
+
+
 {{<code "yaml" "3-6">}}
 asyncapi: '2.0.0'
 id: hello-world-app
 servers:
   - url: kafka.mycompany.com
     protocol: kafka
-    description: This is "My Company" Kafka instance.
+    description: This is an example of a "My Company" Kafka instance.
```

Comment:
This is not an example of a "My Company". It's the actual "My Company" Kafka instance.